site/src/main/java/uk/nhs/digital/ps/sitemaphandlers/DefaultDocumentSitemapHandler.java

```diff
@@ -0,0 +1,112 @@
+package uk.nhs.digital.ps.sitemaphandlers;
```

Comment:
Move out of publication system (ps) package

Reply:
Thanks for the comment. I will move it to the common package